custom-connectors/inQubaJourney/readme.md

```diff
@@ -0,0 +1,28 @@
+
+## inQuba Journey Power Automate Connector
+The inQuba Journey connector allows you to seamlessly integrate with the inQuba Journey API. This will allow you to use existing connectors in power automate to transfer data to inQuba and track your customer journeys.
+
+

```

Comment:
@prinayinq Please refer the template provided for readme.md file and add required sections.

Reply:
The template link brings a 404. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md Please advise.

Reply:
It is https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

You could find it in the templates folder of the repo.